bundles/org.openhab.binding.mqtt.ruuvigateway/README.md

```diff
@@ -0,0 +1,90 @@
+|-----------|-------------------------------------------|----------|---------|
+| `topic`   | MQTT topic containing the gateway payload | Y        | (N/A)   |
+
+## Channels
+
+| Channel ID                | Item Type                | Description                                                              |
+|---------------------------|--------------------------|--------------------------------------------------------------------------|
+| temperature               | Number:Temperature       | The measured temperature                                                 |
+| humidity                  | Number:Dimensionless     | The measured humidity                                                    |
+| pressure                  | Number:Pressure          | The measured air pressure                                                |
+| batteryVoltage            | Number:ElectricPotential | The measured battery voltage                                             |
+| accelerationx             | Number:Acceleration      | The measured acceleration of X                                           |
+| accelerationy             | Number:Acceleration      | The measured acceleration of Y                                           |
+| accelerationz             | Number:Acceleration      | The measured acceleration of Z                                           |
+| txPower                   | Number:Power             | TX power                                                                 |
+| dataFormat                | Number                   | Data format version                                                      |
+| measurementSequenceNumber | Number:Dimensionless     | Measurement sequence number                                              |
+| movementCounter           | Number:Dimensionless     | Movement counter                                                         |
+| rssi                      | Number                   | Received signal (between the Gateway and the sensor) strength indicator  |
+| ts                        | DateTime                 | Timestamp when the message from Bluetooth-sensor was received by Gateway |
+| gwts                      | DateTime                 | Timestamp when the message from Bluetooth-sensor was relayed by Gateway  |
+| gwmac                     | String                   | MAC-address of Ruuvi Gateway                                             |
+
+Note: not all channels are always updated.
+Available fields depend on [Ruuvi Data Format](https://github.com/ruuvi/ruuvi-sensor-protocols).
+At the time of writing (2022-09), most Ruuvi Tags use Ruuvi Data Format 5 out of box.
+
+Some measurements might not make any sense.
+For example, Ruuvi Tag Pro 2in1 does not have a humidity measurement and thus, the humidity data advertised by the sensor is garbage.
+
+## Example
+
+Please note that Thing and Item configuration can be done fully in MainUI. 
```

Comment:
@lolodomo - I remember seeing a comment somewhere to not explicitly mention MainUI, can you advise? Do we usually just refer to the "UI"?

Reply:
OK. Just before you commented I committed a change for "MainUI" -> "the UI" to not have this small comment block the PR from being merged. That should be okay as well. I saw only very few references to "MainUI" when I grep'ed the repo yesterday and many *.md references to "UI".